### DISCUSSION TITLE
Configuring SMTP through Admin page

### DISCUSSION THREAD

**Discussion by ezpl** [hassio-addons/addon-bitwarden#276] [General]
Initially when installing the add-on, I did not configure SMTP. Trying to set it up later, it seems the way to access these settings is in the admin section (/admin). However, this page wants to send me an authentication key by...e-mail. 

Is there a way I can configure SMTP manually? Or any other way to get around this?

**Comment by clementko** [comment]
If you know what you are doing, you can edit the /data/config.json in the docker directly. Be careful with that file because any mistake you make with the file might stop the addon from starting

**Reply by ezpl** [reply]
Thanks. 
I managed to get it working. Should really be available in a better way though 😏
